initfiles/componentfiles/configschema/templates/AddAuthenticateFeature.json

```diff
@@ -0,0 +1,169 @@
+                        "name": "resource",
+                        "value": "DeployWorkunitsAccess"
+                    },
+                    {
+                        "name": "service",
+                        "value": "ws_workunits"
+                    }
+                ]
+            }
+        },
+        {
+            "action": "create",
+            "description": "Add the AuthenticateFeature key to all ESP bindings for ECL Watch",
+            "target_path": "/Environment/Software/EspService[@buildSet=('ws_sql')]/Properties",
+                "data": {
+                "node_type": "AuthenticateFeature",
+                "attributes": [
+                    {
+                        "name": "description",
+                        "value": "Access to deploy workunits"
+                    },
+                    {
+                        "name": "path",
+                        "value": "DeployWorkunitsAccess"
+                    },
+                    {
+                        "name": "resource",
+                        "value": "DeployWorkunitsAccess"
+                    },
+                    {
+                        "name": "service",
+                        "value": "ws_sql"
+                    }
+                ]
+            }
+        },
+        {
+            "action": "find",
+            "description": "Get all of the ECL watch defined service names",
+            "target_path": "/Environment/Software/EspService[@buildSet=('ws_sql')]",
+                "data": {
+                "attributes": [
+                    {
+                        "name": "name",
+                            "save": {
+                            "name": "WsSQLName",
+                            "duplicate_ok": true
+                        }
+                    }
+                ]
+            }
+        },
+        {
+            "action": "create",
+            "description": "Add the AuthenticateFeature key to all ESP bindings for ECL Watch",
+            "target_path": "/Environment/Software/EspProcess/EspBinding[@service=('{{WsSQLName}}')]",
+                "data": {
+                "node_type": "AuthenticateFeature",
+                "attributes": [
+                    {
+                        "name": "description",
+                        "value": "Access to deploy workunits"
+                    },
+                    {
+                        "name": "path",
+                        "value": "DeployWorkunitsAccess"
+                    },
+                    {
+                        "name": "resource",
+                        "value": "DeployWorkunitsAccess"
+                    },
+                    {
+                        "name": "service",
+                        "value": "ws_sql"
+                    }
+                ]
+            }
+        },
+        {
+            "action": "create",
+            "description": "Add the AuthenticateFeature key to all ESP bindings for ECL Watch",
```

Comment:
And, this is adding the feature to the build set entry for the ECL Watch service (not the binding)

Reply:
Changed to service